backend/src/Core/FurnitureShop.Core.Services/CQRS/Mobile/Categories/UpdateCategoryCS.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FurnitureShop.Core.Contracts.Mobile.Categories;
+using FurnitureShop.Core.Domain;
+using FurnitureShop.Core.Services.DataAccess;
+using Microsoft.EntityFrameworkCore;
+
+namespace FurnitureShop.Core.Services.CQRS.Mobile.Categories
+{
+    public class UpdateCategoryCH : ICommandHandler<UpdateCategory>
+    {
+        private readonly Serilog.ILogger logger = Serilog.Log.ForContext<UpdateCategoryCH>();
+        private readonly CoreDbContext dbContext;
+        public UpdateCategoryCH(CoreDbContext dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+
+        public async Task ExecuteAsync(CoreContext context, UpdateCategory command)
+        {
+            var category = await dbContext.Categories.Where(c => c.Id == command.Id).FirstOrDefaultAsync();
```

Comment:
Default is not a null IIRC, please check it and if it's like this change it

Reply:
jeśli kwerendowanym obiektem jest klasa to default jest null